### PR TITLE
feat: add kilt example

### PR DESCRIPTION
**Pull request by kratico**

### CODE REVIEW

File: examples/kilt/did.eg.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * @title Kilt decentralized identifier (DID)
```
Comment (harrysolovay) [inline examples/kilt/did.eg.ts:2]:
```suggestion
 * @title Read Kilt Web3 names and then use them to retrieve DID details
```

**Review by harrysolovay — CHANGES_REQUESTED** [review]
Actually... perhaps `examples/read/kilt.eg.ts` for now?

**Comment by kratico** [comment]
Example to get the first 10 owners
```
import { $didPublicKeyDetails, spiritnet } from "@capi/spiritnet"
import { $, ArrayRune, is } from "capi"
import { equals } from "../../deps/std/bytes.ts"

const didPublicKeyDetails = await spiritnet.Web3Names.Owner
  .entries({ limit: 10 })
  .into(ArrayRune)
  .mapArray((entry) =>
    spiritnet.Did.Did
      .value(entry.access(1, "owner"))
      .unhandle(is(undefined))
      .map(({ authenticationKey, publicKeys }) =>
        [...publicKeys.entries()].find(([k, _v]) => equals(k, authenticationKey))
      )
      .unhandle(is(undefined))
  )
  .run()

console.log("DID public key details:", didPublicKeyDetails)
$.assert($.array($.tuple($.sizedUint8Array(32), $didPublicKeyDetails)), didPublicKeyDetails)

File: examples/read/kilt.eg.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const owner = spiritnet.Web3Names.Owner
+  .entries({ limit: 1 })
+  .into(ValueRune)
+  .access(0, 1, "owner")
+
+const didPublicKeyDetails = await spiritnet.Did.Did
+  .value(owner)
+  .unhandle(is(undefined))
+  .map(({ authenticationKey, publicKeys }) => publicKeys.get(authenticationKey))
+  .unhandle(is(undefined))
```
Comment (harrysolovay) [inline examples/read/kilt.eg.ts:17]: We can probably get rid of this last unhandle

File: examples/read/kilt.eg.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import { $didPublicKeyDetails, spiritnet } from "@capi/spiritnet"
+import { $, is, ValueRune } from "capi"
+
+const owner = spiritnet.Web3Names.Owner
+  .entries({ limit: 1 })
+  .into(ValueRune)
+  .access(0, 1, "owner")
+
+const didPublicKeyDetails = await spiritnet.Did.Did
```
Comment (harrysolovay) [inline examples/read/kilt.eg.ts:13]: Let's just name this "details" (?)

File: examples/read/kilt.eg.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  .access(0, 1, "owner")
+
+/// Retrieve the associated DID public key details.
+const details = await spiritnet.Did.Did
+  .value(owner)
+  .unhandle(is(undefined))
+  .map(({ authenticationKey, publicKeys }) => publicKeys.get(authenticationKey))
+  .run()
+
+/// Ensure `details` is of the expected shape
```
Comment (harrysolovay) [inline examples/read/kilt.eg.ts:23]:
```suggestion
/// Ensure `details` is of the expected shape.
```

**Review by tjjfvi — APPROVED** [review]

**Review by harrysolovay — APPROVED** [review]

**Pull request merged**